guides/createiso/index.html

```diff
@@ -236,6 +236,40 @@ <h2>Part 3: Extracting the WIM</h2>
                 </r-cell>
             </r-grid>
         </div>
+        <div class="hero-sect grey main-breakout">
+            <r-grid columns=2>
+                <r-cell span-s=row>
+                    <h2>Part 4: Converting the WIM</h2>
+                    <p>
+                        There are issues with unofficial install.WIM files made using UUPdump (and possibly other) conversion websites.
+                        In return, after adding .WIM file in WOA-Deployer, it will report an error "Invalid WIM file".
+                    </p>
+                    <p>
+                        To fix this issue, open terminal on Windows with administrator rights and execute this command to convert .WIM file:
+                    </p>
+                    <code>
+                        DISM /Export-Image /SourceImageFile:[original install.wim location] /SourceIndex:1 /DestinationImageFile:[new install.wim location] /CheckIntegrity
```

Comment:
CheckIntegrity only takes more time and is not needed, it should probably be removed

Source Index may not be "1" depending on what ISO file the user wanted to use (it may have more than one edition/index as a consequence), not sure how this should be tackled in the guide